app/repositories/games.py

```
@@ -96,6 +96,18 @@ def update_game_by_id(game_id: int, game_in: GameIn) -> bool:
             return cursor.rowcount > 0
 
 
+def update_game_fields_by_id(game_id: int, fields: dict) -> bool:
+    if not fields:
+        return False
+    with mariadb.connect(**db_config) as conn:
+        with conn.cursor() as cursor:
+            set_clause = ", ".join(f"{k}=?" for k in fields)
+            values = list(fields.values()) + [game_id]
```

Ruff (0.15.12)

[warning] Consider `[*list(fields.values()), game_id]` instead of concatenation

Replace with `[*list(fields.values()), game_id]`

(RUF005)